torchvision/prototype/features/_utils.py

```diff
@@ -0,0 +1,4 @@
+from typing import List, Union
+
+# Same definition as in `functional._utils.py`. Copied here to avoid cyclical dependencies.
+FillType = Union[int, float, List[float], None]
```

Comment:
Alternatively I can move all `functional._utils.py` here and import the other way around.